### PR TITLE
Add encoding model to entity/claim extraction config sections

### PR DESCRIPTION
**Pull request by darthtrevino**
## Description

[Provide a brief description of the changes made in this pull request.]

## Related Issues

[Reference any related issues or tasks that this pull request addresses.]

## Proposed Changes

[List the specific changes made in this pull request.]

## Checklist

- [ ] I have tested these changes locally.
- [ ] I have reviewed the code changes.
- [ ] I have updated the documentation (if necessary).
- [ ] I have added appropriate unit tests (if applicable).

## Additional Notes

[Add any additional notes or context that may be helpful for the reviewer(s).]

### CODE REVIEW

File: docsite/posts/config/env_vars.md [hunk 1]
```diff
@@ -146,12 +146,14 @@ These settings control the data input used by the pipeline. Any settings with a
 | `GRAPHRAG_ENTITY_EXTRACTION_PROMPT_FILE`      | The path (relative to the root) of an entity extraction prompt template text file.         | `str`    | optional             | `None`                                                           |
 | `GRAPHRAG_ENTITY_EXTRACTION_MAX_GLEANINGS`    | The maximum number of redrives (gleanings) to invoke when extracting entities in a loop.   | `int`    | optional             | 1                                                                |
 | `GRAPHRAG_ENTITY_EXTRACTION_ENTITY_TYPES`     | A comma-separated list of entity types to extract.                                         | `str`    | optional             | `organization,person,event,geo`                                  |
+| `GRAPHRAG_ENTITY_EXTRACTION_ENCODING_MODEL`		| The encoding model to use for entity extraction.                                           | `str`    | optional             | `None`                                                   |
```
Comment (natoverse) [inline docsite/posts/config/env_vars.md:149]: Can we clarify that it rolls up to the top config if None?

File: docsite/posts/config/env_vars.md [hunk 2]
```diff
@@ -146,12 +146,14 @@ These settings control the data input used by the pipeline. Any settings with a
...
 | `GRAPHRAG_CLAIM_EXTRACTION_ENABLED`           | Whether claim extraction is enabled for this pipeline.                                     | `bool`   | optional             | `False`                                                          |
 | `GRAPHRAG_CLAIM_EXTRACTION_DESCRIPTION`       | The claim_description prompting argument to utilize.                                       | `string` | optional             | "Any claims or facts that could be relevant to threat analysis." |
 | `GRAPHRAG_CLAIM_EXTRACTION_PROMPT_FILE`       | The claim extraction prompt to utilize.                                                    | `string` | optional             | `None`                                                           |
 | `GRAPHRAG_CLAIM_EXTRACTION_MAX_GLEANINGS`     | The maximum number of redrives (gleanings) to invoke when extracting claims in a loop.     | `int`    | optional             | 1                                                                |
+| `GRAPHRAG_CLAIM_EXTRACTION_ENCODING_MODEL`		| The encoding model to use for claim extraction.                                           | `str`    | optional             | `None`
```
Comment (natoverse) [inline docsite/posts/config/env_vars.md:156]: Same

**Review by natoverse — DISMISSED** [review]

File: docsite/posts/config/env_vars.md (hunk 1 above)
Comment (darthtrevino) [reply docsite/posts/config/env_vars.md]: Good idea, done

File: docsite/posts/config/env_vars.md (hunk 2 above)
Comment (darthtrevino) [reply docsite/posts/config/env_vars.md]: Done

**Review by natoverse — APPROVED** [review]

**Pull request merged**